src/renderer/components/table/virtual-table-header.module.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  display: block;
+
+  tr {
```

Comment:
Implicit styling. Eg. what if a use-case requires styling that is different than this? That would require a more specific class to cascade these stylings, whatever they may be at the time. This makes the styling brittle. It would be better to to pick and choose what kind of styling is required at what point of time, and thus make the intent and responsibility of the styling explicit.

I don't think we